src/_tests/integration/fire-cloud-utils.js

```diff
@@ -0,0 +1,33 @@
+const { findText, waitForNoSpinners, delay } = require('./integration-utils')
+
+
+const fireCloud = () => {
+  const openFireCloudSelector = async page => {
+    return (await page.waitForXPath(`//*[@data-test-id="workspace-selector"]`)).click()
+  }
+
+  const selectWorkSpace = async (page, billingAccount, workSpace) => {
+    await openFireCloudSelector(page)
+    return (await page.waitForXPath(`//ul/li[contains(normalize-space(.),"${billingAccount}/${workSpace}")]`)).click()
+  }
+
+  const findFireCloudClickable = (page, text) => {
+    return page.waitForXPath(`(//a | //*[@role="button"])[contains(normalize-space(.),"${text}") or contains(@data-test-id,"${text}")]`)
+  }
+
+  const click = async (page, text) => {
+    return (await findFireCloudClickable(page, text)).click()
```

Comment:
Either await or return, don't need both. This could be a synchronous function, like the one above it.

Reply:
This line has 2 async things happening, which is why there is both an `await` which is awaiting the promise returned from `findFirecloudClickable()` and then the `return` is returning the promise that comes back from the asynchronous function `click()` called at the end of the line. So I believe it needs both of these?